zerver/actions/message_send.py

```diff
@@ -1455,7 +1515,11 @@ def check_send_message(
         )
     except ZephyrMessageAlreadySentError as e:
         return SentMessageResult(message_id=e.message_id)
-    return do_send_messages([message], mark_as_read=[sender.id] if read_by_sender else [])[0]
```

Comment:
This `message` local variable should be called `message_request`, since it's a `SendMessageRequest`, not a `Message`. Can you do a prep commit fixing that here, and anywhere else the same mistake occurs in this file?